From 901ecc8a8fac3b39f57294a940af6dab561c7f58 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Tue, 10 Dec 2013 12:34:19 +0000
Subject: [PATCH 0501/1076] MANTIS 0029068: CR 006 - Overhead Display Control

second_screen.idl
V0.1

use string in IDL for interface_version


git-svn-id: svn://localhost/SelfServiceCommon/trunk@622 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/second_screen.idl | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/second_screen.idl b/SelfServiceCommon/Massai/idl/Massai/second_screen.idl
index 46e04d6e..978d4b7f 100644
--- a/SelfServiceCommon/Massai/idl/Massai/second_screen.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/second_screen.idl
@@ -1,5 +1,4 @@
 //  purpose:  interface definition for the second screen handler module
-//  version:  0.1
 #ifndef SECOND_SCREEN_IDL
 #define SECOND_SCREEN_IDL
 
@@ -7,6 +6,8 @@
 
 module second_screen
 {
+    const string interface_version = "0.1";
+
     /** Definition of platform running states */
     enum plf_running_state
     {
@@ -30,9 +31,9 @@ module second_screen
     /** Returncodes returned by the second screen module */
     enum rc
     {
-        RC_OK,		       /**< second screen shows configured information for new platform state */
-        RC_ERROR,	       /**< state change failed */
-        RC_NO_2NDSCREEN    /**< no second screen available (chrome not found or no screen present) */
+        RC_OK,           /**< second screen shows configured information for new platform state */
+        RC_ERROR,        /**< state change failed */
+        RC_NO_2NDSCREEN  /**< no second screen available (chrome not found or no screen present) */
     };
     
     interface control_second_screen
-- 
2.41.0.windows.1

